package.json

```diff
@@ -9,14 +9,18 @@
   "devDependencies": {
     "ava": "^3.15.0",
     "elm": "^0.19.1-5",
+    "elm-doc-preview": "^5.0.5",
```

Comment:
I hadn't checked this library out until recently. It's pretty cool.

Reply:
I only discovered it recently, it's a really nice feedback loop when you're working on a library!